Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [workspace]
 resolver = "2"
 
-members = ["extendr-api", "extendr-engine", "extendr-macros"]
+members = ["extendr-api", "extendr-engine", "extendr-macros", "xtask"]
```

Comment:
I think we don't want `xtask` to be part of the workspace. Am I wrong?

Reply:
I don't know, but the CI fails with this error and I'm just too tired to fix this.

```
cargo extendr r-cmd-check --error-on warning --check-dir extendrtests_check
error: package(s) `xtask` not found in workspace `D:\a\extendr\extendr`
```